test/parallel/test-domain-uncaught-exception.js

```diff
@@ -23,7 +23,7 @@ function test1() {
    */
   const d = domain.create();
   d.run(function() {
-    setTimeout(function onTimeout() {
+    setImmediate(function onSetImmediate() {
```

Comment:
For this test, instead of changing the `setTimeout()` to `setImmediate()`, it is better to keep `setTimeout()` but add a duration of `1`. @sam-github is right. This is testing a timer, not an immediate. Changing it to an immediate changes the test. In other words, the `setTimeout()` is not an incidental tool used within the test. It is part of what is actually being tested.

(In @amazingandyyy's defense, I'm the one that suggested changing `setTimeout()` with no duration to `setImmediate()` in the first place. But yeah, that's not right in this test.)